cmd/run.go

```diff
@@ -84,12 +82,20 @@ func SetupRunFlags(cmd *cobra.Command) {
 		- hypersync: Will sync by downloading historical state, and will NOT
 		  download historical blocks. Can only be set if HyperSync is true.`)
 
+	// PoS Validator
+	cmd.PersistentFlags().String("pos-validator-seed", "", "A BIP39 seed phrase used to generate the private key of the "+
+		"Proof of Stake validator. Setting this flag automatically makes the node run as a Proof of Stake Validator.")
+
 	// Mempool
 	cmd.PersistentFlags().Uint64("mempool-backup-time-millis", 30000,
 		"The frequency in milliseconds with which the mempool will persist its state to disk. "+
 			"The default value is 30 seconds, or 30,000 milliseconds.")
 	cmd.PersistentFlags().Uint64("max-mempool-pos-size-bytes", 3000000000,
 		"The maximum size of the PoS mempool in bytes. The default value is 3GB.")
+	cmd.PersistentFlags().Uint64("mempool-fee-estimator-num-mempool-blocks", 50,
```

Comment:
I think we want 1 as the default for mempool blocks and 60 for past blocks. @diamondhands0 for final call on default values here.